client/my-sites/plugins/main.jsx

```diff
@@ -55,7 +55,7 @@ export default React.createClass( {
 	mixins: [ URLSearch, PluginNotices ],
 
 	getInitialState() {
-		return this.getPluginsState( this.props );
+		return Object.assign( this.getPluginsState( this.props ), { bulkManagement: [] } );
```

Comment:
Is the intended goal to have bulk management empty on every load? Right now that is what will happen regardless of what's in `this.getPluginsState( this.props )`

If that is merely a fallback, it should be the first argument to `Object.assign()` so that it would be overwritten by `this.getPluginState()`